projects/fsastorefrontlib/src/cms-components/agent/appointment-scheduling/appointment-scheduling-form.component.html

```diff
@@ -62,14 +62,17 @@
     <cx-form-errors [control]="form.get('description')"></cx-form-errors>
   </div>
   <div class="form-check">
-    <input
-      class="form-check-input"
-      type="checkbox"
-      formControlName="consentGiven"
-    />
-    <span class="form-check-label cx-be-bold">{{
-      'appointmentScheduling.consent' | cxTranslate
-    }}</span>
+    <label for="consentGiven">
```

Comment:
Small fix to enable checkbox to be checked if the label itself is clicked, as on the rest of the site